### ISSUE TITLE
Notification that call was blocked

### ISSUE THREAD

**Issue by jmlich** [jmlich/exphone#3]
There probably should be an notification about "blocked call" to make sure that blocking wasn't unwanted.

**Comment by gbdomubpkm** [comment]
A history of blocked calls can also be interesting.

**Comment by jmlich** [comment]
The call history is a fundamental building block of UI. Do you need detailed history of blocked numbers? It stores number of calls and timestamp of last call.

**Comment by gbdomubpkm** [comment]
Yes, it is a history of blocked calls mentioning the phone number (which is in the address book or not) or blocked hidden call (if it is done), as well as the time when the call was blocked. The calls in the history could be deleted one by one or globally.
This is obviously just a suggestion.

**Comment by jmlich** [comment]
Fixed by 70429feb97dfabf2df54f23b7182f4e71c8234fc
